server/src/main/java/fr/uga/l3miage/tp4/repositories/CandidateRepository.java

```diff
@@ -0,0 +1,16 @@
+package fr.uga.l3miage.tp4.repositories;
+
+import fr.uga.l3miage.tp4.models.CandidateEntity;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import java.time.LocalDate;
+import java.util.Set;
+
+@Repository
+public interface CandidateRepository extends JpaRepository<CandidateEntity, LocalDate> {
+        Set<CandidateEntity> findByExamScore(int score);
```

Comment:
Plusieurs chose ne vont pas ici, déjà on les cherches tous, donc `findAll`, sinon candidat ne prote pas du tous l'attribut ExamScore, donc ça ne peux pas marcher... 
```suggestion
        Set<CandidateEntity> findByExamScore(int score);
```